docs/docs/contributing.md

```diff
@@ -104,6 +144,67 @@ Our testing guidelines regarding fixtures are as follows:
   [loan-fixtures methods](http://www.scalatest.org/user_guide/sharing_fixtures#loanFixtureMethods). Otherwise, prefer
   parameterized case classes or traits.
 
+## Running Integration Tests
+
+### Prerequisites
+
+Running integration tests requires that several components are installed and available locally:
+
+* Mesos (master and agent)
+* Docker
+* Mesos native lib (with `MESOS_NATIVE_JAVA_LIBRARY` set)
+
+Note that the integration test suites launch and use a local, embedded, version of Zookeeper.
+
+#### Installing Mesos
+
+Marathon integration tests require an installation of Mesos in order to run. The Mesos executables, including `mesos`, `mesos-master`, and `mesos-agent` must be included in the `PATH`. Further, the environment variable `MESOS_NATIVE_JAVA_LIBRARY` should be set to the path of `libmesos.dylib` / `libmesos.so`.
+
+##### Installing from source, manually
+
+You can install Mesos from source, manually, by following the instructions [here](http://mesos.apache.org/documentation/latest/building/)
+
+##### Installing from source using mvm.sh
+
+To install Mesos from source, you can use the provided `scripts/mvm.sh` script. Please read the header of the script for pre-requisite steps.
+
+```
+scripts/mvm.sh --latest
+```
+
+This will take 30 minutes or so to install on a modern processor.
+
+##### Installing from Homebrew (OS X, fastest)
+
+You can install a version of Mesos from Homebrew. The Homebrew Mesos version often lags behind the latest version of Mesos. Most of the time, however, this is not a problem.
+
+### Running Tests
+
+Some integration tests are disabled by default (references `WhenEnvSet`). Setting these environment variables will enable them.
+```
+export RUN_DOCKER_INTEGRATION_TESTS=false
+export RUN_MESOS_INTEGRATION_TESTS=false
```

Comment:
Both of these default to true. The text should say something like

> Some integration tests can be disabled by setting specific environment variables. Setting the following will disable tests for docker images run via the Docker containerizer and the Mesos containerizer, respectively.

Reply:
✅